examples/create-inner-asset/assets/master_app.py

```diff
@@ -0,0 +1,75 @@
+from pyteal import *
+
```

Comment:
1. please add a doc comment to what this app is doing, also say that his is demo app and doesn't have security measures. Make similar description in the README file.
2. explain the composition of transactions, this app should be called as a 3rd transaction, right?
3. maybe let's rename to something more meaningful? How about "coordinator.py"?